doc/changelog/07-vernac-commands-and-options/15802-deprecate_backinstance.rst

```diff
@@ -0,0 +1,9 @@
+- **Added:**
+  A deprecation warning that :n:`>` in :cmd:`Class` will in the future also
+  declare a :ref:`coercion <coercions>` as it does when used in
+  :cmd:`Record` commands
```

Comment:
```suggestion
  A deprecation warning that the ``Class >`` syntax or :n:`:>` in the field of
  a :cmd:`Class`, will in the future also   declare a :ref:`coercion <coercions>`
  as it does when used in   :cmd:`Record` commands
```

Or something similar. I would have thought that `:>` is more important to mention than `>` here.